components/service/glean/src/test/java/mozilla/components/service/glean/ping/PingMakerTest.kt

```diff
@@ -27,6 +28,7 @@ class PingMakerTest {
     private val mockApplicationContext = mock(Context::class.java)
 
     @Test
+    @Config(minSdk = 21)
```

Comment:
Since we know that this behavior can be different based on the Android version I wonder whether we should at more API levels here - or at least `24` where we know that `Z` was introduced.

Reply:
Oh, I didn't know about the difference between `sdk` and `minSdk`. This is perfect then! :)